From fae17dc5d279bc6de0ad045c87a6e224a62b3730 Mon Sep 17 00:00:00 2001
From: Benjamin Kaduk <bkaduk@akamai.com>
Date: Tue, 7 Sep 2021 09:20:44 -0700
Subject: [PATCH 52/63] QUIC: remove SSL_get_current_cipher() reference

The QUIC APIs have no need to interact with TLS ciphers, since
QUIC records use different cryptographic protections than TLS ciphers.
---
 doc/man3/SSL_CTX_set_quic_method.pod | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/doc/man3/SSL_CTX_set_quic_method.pod b/doc/man3/SSL_CTX_set_quic_method.pod
index 1ed076c301..f12aa8662b 100644
--- a/doc/man3/SSL_CTX_set_quic_method.pod
+++ b/doc/man3/SSL_CTX_set_quic_method.pod
@@ -209,9 +209,6 @@ only available in the client to server direction. The other secret will be
 NULL. The server acknowledges such data at B<ssl_encryption_application>,
 which will be configured in the same SSL_do_handshake() call.
 
-This function should use SSL_get_current_cipher() to determine the TLS
-cipher suite.
-
 add_handshake_data() adds handshake data to the current flight at the given
 encryption level. It returns one on success and zero on error.
 
-- 
2.42.0

